docs/cover.rst

```diff
@@ -0,0 +1,30 @@
+Cover
+========
+
+Usage: `cover()`
+
+Description
+-----------
+
+This filter is used in gifs to extract the first frame of them as an image to be used as cover.
```

Comment:
I suggest GIFs as a more accurate term (Wikipedia also uses it) and a small rewording:

```suggestion
This filter is used in GIFs to extract their first frame as the image to be used as cover.
```

Reply:
The suggestions were applied. Thank you!